File: Teste/app/src/main/java/teste/com/br/teste/activity/MainActivity.java
package teste.com.br.teste.activity;

import android.app.FragmentManager;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;

import teste.com.br.teste.R;
import teste.com.br.teste.fragment.ChangeFragment;
import teste.com.br.teste.fragment.RepositoryFragment;

public class MainActivity extends AppCompatActivity {

    Toolbar myToolbar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        myToolbar = (Toolbar) findViewById(R.id.toolbar);
        setSupportActionBar(myToolbar);
        ChangeFragment.changeFragment(R.id.conatiner, RepositoryFragment.class, getFragmentManager(), false, null);
   }

    public Toolbar getMyToolbar() {
        return myToolbar;
    }

}

File: app/src/main/java/com/araujoraul/mvvmapp/MainActivity.kt
package com.araujoraul.mvvmapp

import android.os.Bundle
import android.view.Menu
import android.view.MenuItem
import android.widget.Toast
import com.google.android.material.navigation.NavigationView
import androidx.navigation.findNavController
import androidx.navigation.ui.AppBarConfiguration
import androidx.navigation.ui.navigateUp
import androidx.navigation.ui.setupActionBarWithNavController
import androidx.navigation.ui.setupWithNavController
import androidx.drawerlayout.widget.DrawerLayout
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar
import com.araujoraul.mvvmapp.db.ItemDatabase
import com.araujoraul.mvvmapp.ui.cart.CartAdapter
import com.araujoraul.mvvmapp.util.Coroutines

class MainActivity : AppCompatActivity() {

    private lateinit var appBarConfiguration: AppBarConfiguration
    private val toolbar by lazy { findViewById<Toolbar>(R.id.toolbar) }
    private val drawerLayout by lazy { findViewById<DrawerLayout>(R.id.drawer_layout) }
    private val navView by lazy { findViewById<NavigationView>(R.id.nav_view) }
    private val navController by lazy { findNavController(R.id.nav_host_fragment) }



    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        setSupportActionBar(toolbar)
        supportActionBar?.title = getString(R.string.menu_cart)

        appBarConfiguration = AppBarConfiguration(setOf(R.id.nav_cart), drawerLayout)
        setupActionBarWithNavController(navController, appBarConfiguration)
        navView.setupWithNavController(navController)

    }

    override fun onCreateOptionsMenu(menu: Menu): Boolean {
        menuInflater.inflate(R.menu.main, menu)
        return true
    }

    override fun onSupportNavigateUp() = navController.navigateUp(appBarConfiguration) || super.onSupportNavigateUp()

}
